src/main/java/uk/gov/ons/census/casesvc/service/UacService.java

```diff
@@ -126,7 +131,7 @@ public void ingestUacCreatedEvent(ResponseManagementEvent uacCreatedEvent) {
             uacCreatedEvent.getPayload().getUacQidCreated().getUac(),
             uacCreatedEvent.getPayload().getUacQidCreated().getQid());
 
-    saveAndEmitUacUpdatedEvent(uacQidLink);
+    saveAndEmitUacUpdatedEvent(uacQidLink, false);
```

Comment:
Why not just call the method where unreceipted defaults to false, seeing as you went to the trouble of overloading the method?

Reply:
fair point, we were actually going to have a differently named method for the unreceipting, rather than a magic true/false set here